docs/src/views/FAQ.tsx

```diff
@@ -111,6 +111,42 @@ export default () => (
       }
     />
 
+    <Question>
+      What's the difference between {code('Flex')}, {code('Grid')}, {code('Layout')}, {code('Box')}{' '}
+      and {code('Segment')} components?
+    </Question>
+    <Answer
+      content={
+        <div>
+          <p>
+            {code('Flex')}, {code('Grid')} and {code('Layout')} components handle layout aspects.
+            The {code('Flex')} component is for laying out items in one direction, while the{' '}
+            {code('Grid')} component is made for two dimensional layouts. Visit the{' '}
+            {link('Layout guide', '/layout')} page for a detailed comparison between {code('Flex')}
+            and {code('Grid')}. The {code('Layout')} component is now deprecated. Its purpose was
```

Comment:
nit: may suggest to put comments about `Layout` in a separate paragraph, it will make the content easier to read